SIS_import/sis_upload.rb

```diff
@@ -476,11 +513,20 @@ def create_instructor_sandbox_site(zip_file_name, outputFile)
 
 if (upload_error)
 	## check first about the environment variable setting for MAILTO '
-	p "Use the environment variable 'MAILTO' for sending out error messages to #{ENV['MAILTO']}"
+	p "Use the environment variable 'MAILTO' for sending out SIS upload error messages to #{ENV['MAILTO']}"
 	p upload_error
 	## send email to support team with the error message
-	`echo #{upload_error} | mail -s "#{$server} Upload Error" #{ENV['MAILTO']}`
+	`echo #{upload_error} | mail -s "#{$server} SIS Upload Error" #{ENV['MAILTO']}`
 else
+	if ($upload_warnings != "")
+		# mail the upload warning message
+		## check first about the environment variable setting for MAILTO '
+		p "Use the environment variable 'MAILTO' for sending out SIS upload warning messages to #{ENV['MAILTO']}"
+		p $upload_warnings
+		## send email to support team with the error message
+		`echo #{$upload_warnings}  | mail -s "#{$server} SIS Upload Warnings" #{ENV['MAILTO']}`
```

Comment:
I was referring to the status code for mail. It's nice to know if mail has
a problem but if the information is also in the log it's probably not
necessary to start reporting on the status of mail.

On Thu, Jun 25, 2015 at 11:22 AM, Zhen Qian <notifications@github.com>
wrote:

> In SIS_import/sis_upload.rb
> <https://github.com/tl-its-umich-edu/canvas/pull/63#discussion_r33267051>:
>
> >  else
> > +	if ($upload_warnings != "")
> > +		# mail the upload warning message
> > +		## check first about the environment variable setting for MAILTO '
> > +		p "Use the environment variable 'MAILTO' for sending out SIS upload warning messages to #{ENV['MAILTO']}"
> > +		p $upload_warnings
> > +		## send email to support team with the error message
> > +		`echo #{$upload_warnings}  | mail -s "#{$server} SIS Upload Warnings" #{ENV['MAILTO']}`
>
> The upload warning message also gets logged into log files.
>
> What "status code" do you refer here? Status code for email or status code
> for SIS upload?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tl-its-umich-edu/canvas/pull/63/files#r33267051>.
>